platform/marvell-armhf/sonic-platform-nokia/7215/scripts/cpu_wdt.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+
+from sonic_platform.chassis import Chassis
+from sonic_py_common import logger
+import time
+import os
+import signal
+import sys
+
+
+TIMEOUT=170
+KEEPALIVE=55
+sonic_logger = logger.Logger('Watchdog')
+sonic_logger.set_min_log_priority_info()
+time.sleep(60)
+chassis = Chassis()
+watchdog = chassis.get_watchdog()
+
+def stopWdtService(signal, frame):
+    watchdog._disablewatchdog()
+    sonic_logger.log_notice("CPUWDT Disabled: watchdog armed=%s" % watchdog.is_armed() )
+    sys.exit()
+
+def main():
+
+    signal.signal(signal.SIGHUP, signal.SIG_IGN)
+    signal.signal(signal.SIGINT, stopWdtService)
+    signal.signal(signal.SIGTERM, stopWdtService)
```

Comment:
@Pavan-Nokia can we keep the watchdog running during reboot time so that we don't ever end up in a hung situation if kernel hangs during reboot?See https://github.com/sonic-net/sonic-buildimage/blob/master/platform/broadcom/sonic-platform-modules-cel/haliburton/script/cpu_wdt#L50

Reply:
We cannot keep the watchdog active during reboot on the 7215-IXS-T1, the watchdog circuit is reset when the system is rebooted, so we have to arm it again when we come back up after reboot

Reply:
@Pavan-Nokiathen what is the point of enabling wachdog? I understand if the system is hung AFTER watchdog is enabled then it works as expected. But consider a case where system is booting up after reboot and hangs...before watchdog is enabled then system is hung and watchdog cannot bail out

Reply:
@prgeor You are right, we cannot bail out if there is a hang before the watchdog is enabled
 Uboot Does not have support for enabling the watchdog currently, and it is complicated to implement this on existing platform.
The only way we can enable the watchdog here is by using a service in SONiC. and this service will be stopped at some point when the switch is going down and re-enable on boot up.